assets/js/sitemap-definitions.js

```diff
@@ -49,11 +49,11 @@
             return result
         }
 
-        data.options.data['itemData'] = iterator($items)
+        data.options.data['Definition'] = {items: iterator($items)};
```

Comment:
This trades one hacky workaround (direct accessing of a value injected into the POST data) with another (magical reference to the `Definition` form alias). 

The better solution would be to store this data within a data locker input (hidden input in the main template for the formwidget itself) and rely on the existing Form logic to process sending the data in the request.

This solution as is will only work in the specific implementation of this FormWidget on the Definition model's fields.yaml configuration.